stock_picking_operations_multilang/models/stock_warehouse.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 Quartilte Limited (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class Warehouse(models.Model):
+    _inherit = "stock.warehouse"
+
+    @api.model
+    def _lang_get(self):
+        return self.env["res.lang"].get_installed()
+
+    picking_operation_language_option = fields.Selection(
+        [("default", "Default"), ("partner", "Partner"), ("warehouse", "Warehouse")],
+        default="default",
+        required=True,
```

Comment:
To make it a bit simpler.  Please add help text.
```suggestion
        [("partner", "Partner"), ("warehouse", "Warehouse")],
```